spec/features/user_creates_a_result_spec.rb

```diff
@@ -77,20 +60,4 @@
     expect(page).to have_content "Annual survey"
     expect(page).to have_content "92"
   end
-
-  scenario "and it defaults to the same name, description as the assessment" do
```

Comment:
I thought they were, but looking at it, they are not. I can do it more directly with a controller spec, though.